src/mediaelement.js

```diff
@@ -59,7 +59,7 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
     },
 
     isPaused: function () {
-        return this.media.paused;
+        return this.media && this.media.paused;
```

Comment:
If it's called internally, then I guess a check is enough. As for returning `false`, I would expect `isPaused` to return `true` if it's not playing.